sharktank/sharktank/layers/paged_llama_attention_block.py

```diff
@@ -99,8 +97,8 @@ def __init__(
                     matmul_kernel=matmul_kernel,
                 ),
             )
-            self.k_quantizer = self.attn_k.q_output
-            self.v_quantizer = self.attn_v.q_output
+            self.paged_attention.kv_cache.k_quantizer = self.attn_k.q_output
+            self.paged_attention.kv_cache.v_quantizer = self.attn_v.q_output
```

Comment:
Nevermind, this won't work since these are theoretically different per stage, and we are passing around a single PagedAttention object to all of them.